test/components/elements/MenuLinkTest.js

```diff
@@ -1,47 +1,31 @@
 'use strict'
+import React from 'react'
 import { expect } from 'chai'
-import { Link } from 'react-router-dom'
-import MenuItem from 'material-ui/MenuItem'
 import MenuLink from 'components/elements/MenuLink'
+import { shallow } from 'enzyme'
+import toJson from 'enzyme-to-json'
+require('test/helpers/enableSnapshot')
```

Comment:
Or maybe, we `import { setupSnaphot, resetSnapshotConfigBeforeEach } from 'test/helpers/enableSnapshot'`…

Then we can call setupSnapshot here, and resetSnapshotConfigBeforeEach within the context of the decribe block where the snapshot happens… 🤔